pkg/vere/lord.c

```diff
@@ -965,17 +965,16 @@ u3_lord_init(c3_c* pax_c, c3_w wag_w, c3_d key_d[4], u3_lord_cb cb_u)
   //
   {
 #ifdef U3_OS_windows
-    c3_c* arg_c[14];
+    c3_c* arg_c[28];
 #else
-    c3_c* arg_c[13];
+    c3_c* arg_c[26];
```

Comment:
It's fine for the buffer to be oversized, the argument count is determined by the presence of a null-pointer "terminating" the argument array. Just zero-initialize it for consistency.